proto/regen/ecocredit/v1/state.proto

```diff
@@ -285,9 +288,12 @@ message BatchOriginTx {
   // the mint process (e.g. polygon, ethereum, verra).
   string source = 2;
 
-  // note is a reference note for accounting that is passed to an event.
-  string note = 3;
```

Comment:
Note was not intended to be something we stored in state, only passed to events including origin tx.

Reply:
Also questioning whether we need note and unclear on the original intention for how it should be used.